src/main/kotlin/com/chattriggers/ctjs/utils/console/ElementaConsole.kt

```diff
@@ -137,6 +143,28 @@ class ConsoleComponent(private val loader: ILoader?) : WindowScreen(ElementaVers
                 textInput.setText("")
                 scrollToLastInput()
             }
+
+            textInput.onKeyType { _, keyCode ->
```

Comment:
Are the try-catch blocks just for index out of bounds exceptions? If so we should definitely be avoiding that with logic